sdks/python/apache_beam/runners/interactive/interactive_beam.py

```diff
@@ -88,7 +88,22 @@ def capture_duration(self, value):
     """
     self.capture_control._capture_duration = value
 
-  # TODO(BEAM-8335): add capture_size options when they are supported.
+  @property
+  def capture_size(self):
```

Comment:
Shoulds this have "limit" or "max" in its name, or something to indicate that this is an upper limit.

Reply:
Renaming it to `capture_size_limit`.